core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.controller.ts

```diff
@@ -104,15 +107,20 @@ angular.module('oppia').directive('topicLandingPage', [
           };
           ctrl.$onInit = function() {
             pathArray = WindowRef.nativeWindow.location.pathname.split('/');
-            ctrl.subject = pathArray[2];
-            topic = pathArray[3];
+            ctrl.subject = pathArray[1];
+            topic = pathArray[2];
             topicData = TOPIC_LANDING_PAGE_DATA[ctrl.subject][topic];
             landingPageData = topicData.page_data;
             ctrl.topicTitle = topicData.topic_title;
             ctrl.lessons = landingPageData.lessons;
             assetsPathFormat = '/landing/<subject>/<topic>/<file_name>';
-            var pageTitle = 'Learn ' + ctrl.topicTitle + ' - Oppia';
-            PageTitleService.setPageTitle(pageTitle);
+
+            ctrl.pageTitle = (
```

Comment:
Do we need to bind `pageTitle` in the controller?

Reply:
Actually, binding it to the controller makes testing easier. It allows us to access this property while testing whether the page title contains a tagline or not.

Reply:
Don't do this (In general don't add stuff to the main code which is just to help to write test), can we use the PageTitleService in the test?

Reply:
Done! The PageTitleService only has `setPageTitle` function, so I've used `$window` instead.